helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -156,6 +154,23 @@ def _read_group_stage_ids(self, stages, domain, order):
     )
     active = fields.Boolean(default=True)
 
+    @api.model
+    def default_get(self, fields):
+        defaults = super().default_get(fields)
+        company_id = defaults.get("company_id") or self.env.company.id
+        if not defaults.get("user_id"):
```

Comment:
To avoid doing it if it’s not necessary.
```suggestion
        if "user_id" in fields and not defaults.get("user_id"):
```